src/common/cache.py

```diff
@@ -90,3 +90,8 @@ def set_messages(self, entry_name, messages):
         with open(export_file_path, "w") as f:
             for msg in messages:
                 f.write(f"{json.dumps(msg.to_dict(serialize_datetimes_to_str=True))}\n")
+
+    def delete_file(self, entry_name):
```

Comment:
I also think use of the file system is arguably implementation specific, so suggest making delete_file private and adding clear_timestamp(self, entry_name) too.